packages/datadog-plugin-next/test/next.config.js

```diff
@@ -2,5 +2,14 @@ module.exports = {
   eslint: {
     ignoreDuringBuilds: true
   },
-  output: 'standalone'
+  output: 'standalone',
+  webpack: (config, { webpack }) => {
+    config.plugins.push(
+      new webpack.NormalModuleReplacementPlugin(/^node:/, resource => {
+        resource.request = resource.request.replace(/^node:/, '')
+      })
```

Comment:
Is this _removing_ use of the `node:` prefix? Do we not _want_ to verify that works? Or is that tested elsewhere?

Reply:
Ayan had asked me to look into this for Next due to build errors without it - basically some version's don't play nice with `node:` prefixes at all, and a common solution I found from other's issues was using this `webpack` option in the config file for Next. However, I can confirm that this only happens on the lowest version we test on the v4 line (v11.0.1 of Next.js), and every other version works without this change. I can push up a change to try and just limit this workaround for only that version, if needed.

Reply:
Would customers need to make the same change to remove the node: prefix for lower versions of next to get it to work  with our tracer?

Reply:
If so I'd rather find a workaround where we don't use node:* modules for our instrumentation of next, we could hop on a call to discuss if the above scenario is the case^?

Reply:
I think it's fine to land as-is then, but with a TODO comment explaining that.

Reply:
Also reverted the change that caused this error, i believe it's an error in the bundler that occurs with node: prefix packages and so the node:net instrumentation always requires 'dns' rather then requiring 'dns' or 'node:dns' based on if 'net' or 'node:net' is being ran because the presence of even a require with 'node:*' causes the bundler to crash